tools/image_processing/imagej2/imagej2_bunwarpj_compare_elastic/imagej2_bunwarpj_compare_elastic.xml

```diff
@@ -0,0 +1,74 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<tool id="imagej2_bunwarpj_align" name="Compare two elastic deformations" version="1.0.0">
+    <description>by warping index with bUnwarpJ</description>
+    <macros>
+        <import>imagej2_macros.xml</import>
+    </macros>
+    <expand macro="fiji_headless_requirements" />
+    <command>
+<![CDATA[
+    python $__tool_directory__/imagej2_bunwarpj_compare_elastic.py
+    --target_image "$target_image"
+    --target_image_format $target_image.ext
+    --source_image "$source_image"
+    --source_image_format $source_image.ext
+    --target_transformation "$target_transformation"
+    --source_transformation "$source_transformation"
+    --max_heap_size_type $set_max_heap_size.max_heap_size_type
+    --max_heap_size $set_max_heap_size.max_heap_size
```

Comment:
And because macros are cool, you could also put the cheetah corresponding to `max_heap_size_type_conditional` into a macro token.